hello-world/nodejs-cli-app/start-instance.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Sample code to start the given instance based on instance OCID and region.
+ * 
+ * Sample Output:
+ * (Prints the instance details)
+ */
+
+const common = require("oci-common");
+const core = require("oci-core");
+
+const adp = new common.ConfigFileAuthenticationDetailsProvider();
+
+const region = common.Region.US_ASHBURN_1;
+const instanceId = "<instance-ocid>"; // Update Instance OCID
```

Comment:
Could the instanceId be populated via an environment variable and/or command-line parameter?

Reply:
Yes. taken as command line argument